app/lib/api_error/level_of_authentication_too_low.rb

```diff
@@ -0,0 +1,27 @@
+module ApiError
+  class LevelOfAuthenticationTooLow < ApiError::Base
+    def initialize(attributes:, needed_level_of_authentication:)
+      @attributes = attributes
+      @needed_level_of_authentication = LevelOfAuthentication.integer_to_name needed_level_of_authentication
+    end
+
+    def status_code
+      :forbidden
+    end
+
+    def detail
+      I18n.t(
+        "errors.level_of_authentication_too_low.detail",
+        attributes: @attributes.join(", "),
+        needed_level_of_authentication: @needed_level_of_authentication,
+      )
+    end
+
+    def extra_detail
+      {
+        attributes: @attributes,
```

Comment:
When would I want this one, and when would I want the comma separated string from L14?

Reply:
The `detail` string is human readable, the `extra_detail` is machine readable